homeassistant/components/flipr/sensor.py

```diff
@@ -0,0 +1,86 @@
+"""Sensor platform for the Flipr's pool_sensor."""
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_TIMESTAMP,
+    TEMP_CELSIUS,
+)
+from homeassistant.helpers.entity import Entity
+
+from . import FliprEntity
+from .const import ATTRIBUTION, CONF_FLIPR_ID, DOMAIN
+
+SENSORS = {
+    "chlorine": {
+        "unit": "mV",
+        "icon": "mdi:pool",
+        "name": "Chlorine",
+        "device_class": None,
+    },
+    "ph": {"unit": "ph", "icon": "mdi:pool", "name": "PH", "device_class": None},
+    "temperature": {
+        "unit": TEMP_CELSIUS,
+        "icon": "mdi:coolant-temperature",
+        "name": "Water Temp",
+        "device_class": DEVICE_CLASS_TEMPERATURE,
```

Comment:
Either set a device class or icon, not both.

Reply:
OK, correction done via a new commit and push.